src/Node.js

```diff
@@ -0,0 +1,48 @@
+  // ### DOM Node Shim
+  //
+  // This represents a superficial form of a DOM node which contains most of
+  // the properties that a regular DOM node would contain, but only the ones
+  // needed by slowparse.
+module.exports = (function(){
+  "use strict";
+
+  function Node(nodeType, nodeName, nodeValue, namespaceURI) {
+    this.nodeType = nodeType || null;
+    this.nodeName = nodeName || null;
+    this.nodeValue = nodeValue || null;
+    this.namespaceURI = namespaceURI || null;
```

Comment:
this is pretty much never relevant, but it's a string, so let's make it `namespaceURI || ""`